### PR TITLE
After image changed sucessufully

### LINKED ISSUE #1580: [Website] enhance before and after screenshot
### Current Behavior
On the homepage, there is a before and after section. The after screenshot is not the best.

image

### Desired Behavior
Replace the after screenshot with a more intriguing version, like this one:

image

Crop the image as needed.

---
<h4>Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources</h4>

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by surajvast1**
**Description**

This PR fixes #1580 

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.
 

image

### CODE REVIEW

**Comment by surajvast1** [comment]
Thanks for giving me this task. Let me know if anything changes—I'm here to help.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Thanks. Try again. You've changed the Kuberentes blue to an off purple color. Retain the same coloring.

**Comment by surajvast1** [comment]
image

I believe the alternative image is much better.
  I belive this is much better image . 

The "before and after" button in the current code exhibits an issue: the "before" button doesn't hide properly after the slider is moved over, and there are functionality concerns when hovering from outside. Would you like me to address this within the current task, or a new issue will be created for it?

**Comment by vishalvivekm** [comment]
@surajvast1
Today, we've our weekly Websites meeting at 6:30 PM IST : https://meet.layer5.io/websites.
Let's get feedback on your work in today's call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by surajvast1** [comment]
Hello @vishalvivekm and @leecalcote , I wanted to inform you that the image and container work for the issue is completed. If there's anything else I can assist with, please let me know. Otherwise, kindly consider merging this issue. Thank you!

File: images/gitops-snapshot.png
[the remark this answers is not in the document]
Comment (surajvast1) [reply images/gitops-snapshot.png:1]: "As we discussed in the meeting, the reviews indicate that the image appears accurate. However, if you feel that adjustments are needed,  ill do it now thanks

[the remark this answers is not in the document]
Comment (surajvast1) [reply images/gitops-snapshot.png:1]: The given image isn't clear. I'll quickly fetch a new one and make the necessary updates.

[the remark this answers is not in the document]
Comment (leecalcote) [reply images/gitops-snapshot.png:1]: The quality of it is crap. The coloration is off. And, the fact that you’re taking an entire screenshot is wrong as well.

**Comment by surajvast1** [comment]
image

**Comment by surajvast1** [comment]
image

**Pull request closed without merging**